### DISCUSSION TITLE
GD Warning: PNG image support has been disabled

### DISCUSSION THREAD

**Discussion by davide98776** [libgd/libgd#859] [General]
Hi

I have a problem with libgd and its PNG support.

I have installed libgd with the command:
sudo apt install libgd-dev

and libpng with the command
sudo apt install libpng-dev

but when i try to create a PNG image this error occurs:
GD Warning: PNG image support has been disabled

how can I solve?
thanks in advance

**Comment by bferguson3** [comment]
Same issue on M1 mac a year and a half later.

**Comment by cmb69** [comment]
I don't think this error messages comes from libgd itself. In which context do you use GD (i.e. which programming language or other library)?
